graphql/mappers.js

```diff
@@ -158,6 +162,54 @@ exports.mapLobbyGroup = (raw, t) => {
   return lobbyGroup
 }
 
+const branchIdPrefix = exports.branchIdPrefix = 'Branch-'
+exports.mapBranch = (raw, t) => {
+
+    const connections = () => {
+    const lobbygroups = raw.interessengruppe.map(connection => ({
+      from: branch,
+      vias: [],
+      to: {
+        id: `${lobbyGroupIdPrefix}${connection.id}`,
+        name: connection.name
+      },
+      group: t('connections/lobbyGroup')
+    }))
+    const organisations = raw.organisationen.map(connection => ({
+      from: branch,
+      vias: [],
+      to: {
+        id: `${orgIdPrefix}${connection.id}`,
+        name: connection.name
+      },
+      group: connection.interessengruppe
```

Comment:
```suggestion
      group: t('connections/organisation')
```

Let's just group all organisations into one. Because `interessengruppe2` and `interessengruppe3` isn't handled—leading to incomplete views and the user should navigate to the lobby group if they want details for a specific group. There they can see a complete list and also parliamentarians by party. Later on we could add parliamentarians grouped by party here too.

Reply:
Hm, I'm not sure if organisations are better.

Rationale:
an organisation refers to lobbygroup from there detail head
a lobbygroup refers to branch from there detail head

So, I would expect to see lobbygroups in a view of branches.
I suggest
1. either we remove the organisations from the list or
2. we keep it as in the pull request.

Reply:
I'd recommend 1.

Otherwise we have mismatches and duplications that become hard to explain:
<img width="1895" alt="Screenshot 2020-05-02 at 19 50 06" src="https://user-images.githubusercontent.com/410211/80871730-3df6bd00-8cae-11ea-8aab-4350fb089e52.png">

And then work on adding parliamentarians directly to branches as soon as possible – that would give a nice overview of e.g. «Aussenpolitik players».

Reply:
OK, good idea.
I'll change it directly to the solution with Parliamentarians.

Reply:
Hm, how should I group the parliamentarian.

1. branch -> lobbygroup -> parliamentarian
   having no "X Lobbygruppen" "connection"
2. branch -> party -> parliamentarian (as for lobbygroups)
   having a Lobbygruppen "bubble": branch -> X Lobbygrupppen -> lobbygroups

I would rather go with solution 2 as it is similar to lobbygroups.
What do you think?

Reply:
I like 2 too

Reply:
I've implemented 2. Parliamentarians are grouped by the party.
Though all parties mostly appear.